src/content/docs/en/pages/guides/edge-storage/use-s3-protocol/index.mdx

```diff
@@ -104,6 +104,26 @@ You can save your settings into an `.s3cfg` file if needed and modify them again
 
 ---
 
+## s3cmd commands to manage objects on Edge Storage
+
+Once you've configured access to the bucket through s3cmd, you can execute operations to manage your objects on Edge Storage using the available commands.
+
+| Scope | Description | 
+|---|---|
+| `s3cmd ls` | List all your buckets |
+| `s3cmd mb s3://my-new-bucket-name` | Create a new bucket. Bucket names must be unique and best practices for naming buckets include specifying what types of objects are stored and the type of permissions for the objects |
+| `s3cmd ls s3://my-new-bucket-name` | List the contents of the bucket |
+| `s3cmd put file.xml s3://my-new-bucket-name/file.xml` | Upload a file into the bucket |
+| `​​s3cmd get s3://my-new-bucket-name/file.xml file-2.xml` | Retrieve the file back and verify that it hasn't been corrupted |
+| `s3cmd del s3://my-new-bucket-name/addrbook.xml s3://my-new-bucket-name/storage.jpg` | Delete the object |
+| `s3cmd rb s3://logix.cz-test` | Remove the bucket. Only empty buckets can be removed |
```

Comment:
It is important to inform the user, that files deleted within the past 24 hours will block the deletion of the entire bucket.

Reply:
A note was added in this section.